tests/ssg_test_suite/virt.py

```diff
@@ -9,9 +9,6 @@
 
 logging.getLogger(__name__).addHandler(logging.NullHandler())
 
-# filled in by connectDomain function
-snapshots = None
-
 
 class SnapshotStack(object):
```

Comment:
Yeah, it was my intention to make it singleton. But then I realized there is no point in it - changed it to something a bit nicer (I hope :) )